tests/simple_fopen.c

```diff
@@ -8,31 +8,43 @@ void
 by_fopen()
 {
     FILE *fd = fopen("tmp", "w");
-    fprintf(fd, "Hi\n");
+    if (!fd)
+    {
+        err(1, "Error in fopen: ");
+    }
     fclose(fd);
 }
 
 void
 by_syscall()
 {
+    int fd = syscall(5, "tmp", O_CREAT); // open
```

Comment:
Are there named constants for syscalls?

Reply:
Yup; fixed in d91f5e82634dbb57346d78c2588be34059199e04.